.github/workflows/coverity.yml

```diff
@@ -0,0 +1,41 @@
+name: Coverity
+
+on:
+  pull_request:
+    branches:
+      - "*"
+  push:
+    branches:
+      - main
+
+jobs:
+  scan:
+    name: "coverity scan"
+    runs-on: ubuntu-latest
+    steps:
+      - name: Clone repository
+        uses: actions/checkout@v2
+
+      - name: Run coverify scan script
+        env:
+          COVERITY_SCAN_PROJECT_NAME: ${{ secrets.COVERITY_SCAN_PROJECT_NAME }}
+          COVERITY_SCAN_TOKEN: ${{ secrets.COVERITY_SCAN_TOKEN }}
+          COVERITY_EMAIL: ${{ secrets.COVERITY_EMAIL }}
```

Comment:
@Gundersanne which email address is this ? Should that be e team-wide address ?

Reply:
It should be an email configured in the coverity project afaik. I'm not 100% sure which I added, might have been mine actually. Problem is it needs an actual verified email account, a team-wide one would be better yea.